script.js

```diff
@@ -1 +1,123 @@
+    const plusLi = document.createElement('li');
+    const delBtn = document.createElement('img');
+
+    //const img=document.createElement("img");
+    delBtn.src="img/bin.png";
+    //delBtn.innerHTML="$";
+
+    //delBtn.append(img);
+    delBtn.addEventListener("click", deleteToDo);
+    const newId=toDos.length+1;
+    //plusLi.innerHTML = input.value;
+    const inputText=document.createTextNode(input.value);
+    plusLi.id=newId;
+
+    const span = document.createElement("span");
+    span.addEventListener("click", toggleToDo);
+    //span.innerText=text;
+    span.appendChild(inputText);
+    plusLi.appendChild(span);
+    plusLi.appendChild(delBtn);
+    delBtn.hidden=true;
+    plusLi.addEventListener("mouseover",function(){
+        plusLi.childNodes[1].hidden=false;
+    });
+
+    waitList.appendChild(plusLi);
+    countTodos();
+
+    const toDoObj = {
+        text: text,
+        id: newId
+    };
+    //saveToDos();
+    toDos.push(toDoObj);
+}
+
+function toggleToDo(event){
+    let span = event.target;
+    let li = span.parentNode;
+    //if(waitList.has)
+    // waitList.removeChild(li);
+    span.removeEventListener("click", toggleToDo);
+    let completedLi = li.cloneNode(true);
+    li.remove();
+    // completeList.appendChild(li);
+    
+    completedLi.childNodes[0].addEventListener("click", toggleUp);
+    completedLi.childNodes[1].addEventListener("click", deleteToDo);
+    completedLi.childNodes[1].hidden=true;
+    completedLi.addEventListener("mouseover",function(){
+        completedLi.childNodes[1].hidden=false;
+    });
+    completeList.appendChild(completedLi);
+    countTodos();
+}
+
+function toggleUp(event){
+    let span=event.target;
+    let li=span.parentNode;
+    span.removeEventListener("click", toggleUp);
+    let incompletedLi = li.cloneNode(true);
+    incompletedLi.childNodes[0].addEventListener("click", toggleToDo);
+    incompletedLi.childNodes[1].addEventListener("click", deleteToDo);
+    incompletedLi.childNodes[1].hidden=true;
+    incompletedLi.addEventListener("mouseover",function(){
+        incompletedLi.childNodes[1].hidden=false;
+    });
+    waitList.appendChild(incompletedLi);
+    li.remove();
+    countTodos();
+}
+
+function deleteToDo(event){
+    console.log("delete");
+    let btn = event.target;
+    let li = btn.parentNode;
+    li.parentNode.removeChild(li);
+    countTodos();
+}
+
+function countTodos() {
+    let completedCount = document.querySelectorAll("ul.wait-list li").length;
+    let incompletedCount = document.querySelectorAll("ul.complete-list li").length;
+    document.getElementById("wait-count").innerHTML = "(" + completedCount.toString() + ")"
+    document.getElementById("complete-count").innerHTML = "(" + incompletedCount.toString() + ")";
+}
+
+function handleSubmit(event){
+    event.preventDefault();
+    const currentValue=input.value;
+    paintToDo(currentValue);
+    input.value="";
+}
+function init(){
+    loadToDos();
+    countTodos();
```

Comment:
대문자 규칙에 대한 정답은 컨벤션이라서 특별히 정해진 바는 없지만,
하나의 프로젝트에서 서로 다른 컨벤션을 혼용하게 되면 일관성이 떨어져서 여러가지 문제들이 생길 수 있겠죵?

```suggestion
    loadToDos();
    countToDos();
```

이런식으로 네이밍 규칙을 맞춰주세요~!